<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="glScene" name="Scene_widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1200</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GenStart">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>630</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>630</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Monte-Carlo Step</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_ForStep">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>630</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_MC_StepsCount">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>630</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Animation">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>630</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Diffuzia Start/Stop </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Count of M-C Steps:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>640</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Current step:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GenStart_1000">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>630</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Start 1000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GenStart_Ogran">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>630</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate Finit Source</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>621</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>621</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Concentration">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>390</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QChartViewer" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>10</y>
        <width>550</width>
        <height>350</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>440</y>
        <width>481</width>
        <height>19</height>
       </rect>
      </property>
      <property name="maximum">
       <number>180</number>
      </property>
      <property name="value">
       <number>99</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>glScene</class>
   <extends>QWidget</extends>
   <header>glscene.h</header>
   <container>1</container>
   <slots>
    <slot>GenerateButton()</slot>
    <slot>slot_TriangulationButton()</slot>
    <slot>slot_CountOfPoints()</slot>
    <slot>slot_VerleButton()</slot>
    <slot>slot_Slider()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QChartViewer</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
